### ISSUE TITLE
Show Examples of Common Enviroment Variables for the major CSPs for Terraform Cloud Workspace

### ISSUE THREAD

**Issue by omenking** [hashicorp/terraform-website#1829]
I am creating a beginner video showing developers how to deploy a VM to all three major providers
- AWS
- Azure
- GCP

When using Terraform Cloud as the backend, you need to provide environment variables for for the CSPs credentials.
Having examples directly on this page would be very useful.

https://www.terraform.io/docs/cloud/workspaces/variables.html

**Comment by omenking** [comment]
Right now in the provider docs I'm trying to figure out what I need to supply as Env vars to Terraform Cloud
![image](https://user-images.githubusercontent.com/7776/130117038-c5d0b4da-3540-4d80-b639-6edb288d4ecd.png)

**Comment by laurapacilio** [comment]
Hi @omenking Thank you so much for submitting this! I did a few things to help alleviate this issue:
- I reworked the variables page a bit to talk more specifically about environment variables and link to the specific places in the Terraform Registry Provider docs for AWS, Azure, and GCP where folks can find the right ones.
- I also worked with the GCP provider team to add clarity to their docs about the right environment variables to set for GCP - it's a bit different than when you're not using Terraform Cloud. 

The PR I put up is linked in this issue if you want to check it out. But I just wanted to let you know what we did to address your concern! :-)
